doc/thirdPartyRunPlugins.md

```diff
@@ -0,0 +1,35 @@
+# Third-Party Run Plugins
+
+Collection of third-party Run plugins created by the community that aren't distributed with PowerToys.
+
+## Installing / Upgrading
+
+The recommended path for the plugins is `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`.
+
+- Close PowerToys
+- Copy the plugin folder to `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`
+- Delete previous version in case of upgrade
+- Open PowerToys
+
+Refer to plugin instruction for more details.
+
+## Disclaimer
+
+The following plugins are created and maintained by third-party and not officially supported or recognized by Microsoft.  
+Users install and use these plugins at their own risk, and Microsoft takes no responsibility for their functionality or any potential issues they may cause.  
+For assistance, please contact the respective plugin developers directly.
+
+## Plugins
+
+| Plugin | Description |
```

Comment:
| Plugin | Author | Description |  

why: I like the idea of being sure we directly give author's credit.  It also directly states this is not a built-in plugin.